journal/modernized/05-chapter-4.adoc

```diff
@@ -280,12 +318,19 @@ Never shall your memory be effaced from my soul, I shall not cease to love you;
 it will be a comfort to my weary soul,
 if sometimes you will make me joyful by imparting a few lines of love,
 and nothing but death will be able to prevent my answering your letters.
-In love I embrace you in my heart, as your ever loving brother,
+In love I embrace you in my heart, as
+
+[.signed-section-closing]
+Your ever loving brother,
 
+[.signed-section-signature]
 Marconnay.
 
+[.signed-section-context-close]
 Berlin, 22nd October, 1796.]
 
+--
+
 Returning now to the Journal:
```

Comment:
I think this phrase "Returning to the Journal" would look best if we put the [.offset] tag above it. That will keep it from being indented, and then put a space below it before the journal picks back up.